pymemcache/client/base.py

```diff
@@ -82,10 +82,10 @@ def _parse_hex(value):
 
 def _check_key(key, key_prefix=b''):
     """Checks key and add key_prefix."""
-    if isinstance(key, six.text_type):
+    if isinstance(key, six.text_type) or isinstance(key, six.string_types):
```

Comment:
is this a possible thing? I like it, it's more elegant. It's slightly less obvious to name than `allowed_key_types`, since it's only dealing with strings. I believe you can have booleans and bytes (and then there's the whole python2/3 thing for bytes)